client/pom.xml

```diff
@@ -13,7 +13,8 @@
 	See the License for the specific language governing permissions and
 	limitations under the License.
 -->
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
```

Comment:
This commit has altered 1200+ lines that are not related to the purpose of your PR, losing valuable history, and making it hard to fully read your PR. Please only commit lines that are related to your PR. Where reformatting is necessary to appease our Checkstyle rules, please just fix the issues you've introduced yourself - please don't try to reformat unrelated lines and unrelated files from across the project.